src/main/webapp/resources/js/pages/admin/components/statistics/TimePeriodSelect.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Form, Select } from "antd";
+
+/**
+ * React component for updating the time period for statistics.
+ * @param {string} [name] - form name value
+ * @param {function} [onChange] - called when the component changes value
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function TimePeriodSelect({ onChange = () => {} }) {
+  return (
+    <Form.Item
+        label ={
+          <span>
+            Time Period
+          </span>
+        }
+      name="time-period"
+    >
+      <Select onChange={onChange} className={"t-time-period"} style={{width: 120}}>
+        <Select.Option value={1}>
```

Comment:
I am wondering if this should not also be a radio.group buttons?  Not to many options and easy to click.  Thoughts?

Reply:
I tried it out and it looks good with the radio group buttons

Reply:
Updated to radio group buttons in 567cdb75bcedd68dc9d0811b02db3369cca719e4